app/main/helpers/validation_tools.py

```diff
@@ -1,31 +1,42 @@
 import os.path
-import time
-import shutil
 
+try:
+    import urlparse
+except ImportError:
+    import urllib.parse as urlparse
```

Comment:
This seems odd - shouldn't installing the requirements ensure that we can import everything we need? i.e. so we shouldn't get ImportErrors?